bam_consensus.c

```diff
@@ -1466,7 +2315,18 @@ static void usage_exit(FILE *fp, int exit_status) {
     fprintf(fp, "      --high-MQ INT     Cap maximum mapping quality [60]\n");
     fprintf(fp, "      --P-het FLOAT     Probability of heterozygous site[%.1e]\n",
             P_HET);
+    fprintf(fp, "      --P-indel FLOAT   Probability of indel sites[%.1e]\n",
+            P_INDEL);
+    fprintf(fp, "      --het-scale FLOAT Heterozygous SNP probability multiplier[%.1e]\n",
+            P_HET_SCALE);
     fprintf(fp, "  -p, --homopoly-fix    Spread low-qual bases to both ends of homopolymers\n");
+    fprintf(fp, "      --homopoly-score FLOAT\n"
+                "                        Qual fraction adjustment for -p option [%g]\n", P_HOMOPOLY);
+    fprintf(fp, "  -t, --qual-calibration FILE / @config (see man page)\n");
```

Comment:
Before we bake this in, do we want to use `FILE / @config` or `@FILE / config`?  The latter is more consistent with the `@FILE` syntax used in [bcftool's expressions](https://www.htslib.org/doc/bcftools.html#expressions) (although I don't think it uses `@FILE` anywhere else).  Other tools that work this way include [curl](https://curl.se/docs/manpage.html#-d) and [javac](https://docs.oracle.com/javase/9/tools/javac.htm#GUID-AEEC9F07-CB49-4E96-8BC7-BCC2C7F725C9__STANDARDOPTIONSFORJAVAC-7D3D9CC2).  Unfortunately I haven't worked out a good way of searching for tools that might go the other way to find out how many there might be.

Reply:
Good question.  We could reverse the syntax I guess, but I'm not sure what the best approach is.

My thought was the primary option is a calibration file and this is what most people would do.  Given the inbuilt tables are basically only there to be used by the `-X profile` canned configurations, it's unlikely people wish to refer to those directly outside of a `-X` option.  They're more or less there just for completeness sake.  (Then again, I'm not that convinced people are going to specify external files either, and again it's more for development of new profiles than much else.)

Bcftools doesn't explain what `@filename` does.  Is it essentially the equivalent of an inline `` `cat filename` `` within an expression?  Similarly @ is working that way for curl and javac.  If so then it's not really the same as what we're doing here, as we're not specifying the calibration data in the command line itself and it's not a synonym for backquoting to load in data and paste in-line.

Reply:
I'm not entirely clear about `bcftools`, but it looks like it allows you to create a set of rules that match a set of samples in a file.  It doesn't just read an expression from a file.  Curl uses it to indicate that the content for the given parameter should come from a file, e.g. `-F profile=@portrait.jpg` makes it do a POST with a file upload.  The java one, on the other hand, reads options from the file, much as if you'd put them on the command-line.  So it's not that standardised.

I guess we can run with the way this currently works, but you can handle any complaints that it doesn't work in the same way as some other tools 😄 .

Reply:
Well I could be persuaded the other way, but won't we then also get complaints that it's the only FILE argument that needs an @ symbol?

I think either way is a bit weird tbh.  Maybe it needs a different symbol?

Reply:
Thinking more, although there is absolutely no real standard between the tools you mentioned with what they do with an `@file`, the one common thread they have is that the `@` means "_do something with this file instead of the normal CLI parameter_".

Therefore I think there is a decision here.

1. Is the "normal" parameter the internal baked-in named calibration table with the file being the exception?  If so the usage needs to not state "FILE", and we can have `@file` as an exception.

2. Is the "normal" parameter the filename mode, in which case the exception shouldn't be using the `@` symbol as that's typically associated with filenames.  Maybe `#` or `=` or something else that's unlikely to start a filename.  I don't know of any commonly used mechanisms for that.